content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+We want to thank all of the contributors and mentors for all of their hard work, as none of this would be possible without them.
+
+link:https://hacktoberfest.com/[Hacktoberfest] is an event to highlight and celebrate open-source projects like Jenkins.
+The main idea of the event is contributing to and improving open-source software.
+People are invited to submit pull requests to contribute to plugins, provide documentation, create translations, and review code among other things, to engage with open-source software. 
+Many products and services are based on open-source software, so giving back in this way continues the cycle of development.
+
+For link:https://www.jenkins.io/events/hacktoberfest/faq/[Jenkins], Hacktoberfest is an opportunity to collaborate with and expand the community, while learning and sharing with all experience levels.
+These contributions are what keeps Jenkins moving forward, so having an opportunity to celebrate both the project and the people is amazing.
+
+== Hacktoberfest results
+
+During October, the Jenkins project received over 613 pull requests from 117 different contributors.
+All of these helped to improve Jenkins, create new resources, and review the needs of Jenkins accordingly.
+From these, 531 pull requests are completed, meaning they have been merged or flagged as "hacktoberfest-approved".
+They were submitted by 95 different contributors, with 42 qualifying for link:https://hacktoberfest.com/participation/#contributors[swag] with Jenkins contributions alone!
+A total of 1,183 manually created pull requests were submitted during October in the jenkins-ci and jenkins-infra organizations.
+Overall, this appears to be a 10 to 15 percent increase in the average number of monthly submissions.
+With all of this in mind, we want to highlight some of the new and returning Jenkins contributors, including their experiences and advice for other users.
+
+== New Jenkins contributors
+
+First, we'd like to share some insight from link:https://github.com/Jagrutiti[Jagruti Tiwari]. 
+Jagruti is Based in Mumbai, India and active in various areas of open source.
+This immersion is the main component of their strategy to explore and learn new things.
+They are not afraid of challenges and have received assistance from the friendly, open-source communities. 
+They are inspired by thought leaders such as Ankita Tripathi.
+Jagruti's contributions to Jenkins during Hacktoberfest includes documentation, code examples, and testing.
+They have also provided reviews for other community members, which supports the open-source spirit.
+
+We also want to welcome link:https://github.com/highflyer910[Thea Mushambadze].
+Thea created three new logos: link:https://www.jenkins.io/artwork/#turkey[Turkey], link:https://www.jenkins.io/artwork/#georgia[Georgia], and a link:https://www.jenkins.io/artwork/#nerd[Jenkins nerd] themed logo.
```

Comment:
Re-reading my notes, her first name is Teona. And she is not a real new commer. I forwarded you her mail.